hydroengine_service/datasets/dataset_visualization_parameters.json

```diff
@@ -218,5 +218,75 @@
 			"wind_speed": ["#172313", "#144b2a", "#187328", "#5f920c", "#aaac20", "#e1cd73", "#fffdcd"],
 			"wind_direction": ["#a8780d", "#d74957", "#d02fd0", "#7d73f0", "#1e93a8", "#359943", "#a8780d"]
         }
-	}
+	},
+    "projects/dgds-gee/gtsm/waterlevel_return_period": {
+        "type": "Image",
+		"bandNames": {
+            "waterlevel_2": "waterlevel_2",
+            "waterlevel_5": "waterlevel_5",
+            "waterlevel_10": "waterlevel_10",
+            "waterlevel_25": "waterlevel_25",
+            "waterlevel_50": "waterlevel_50",
+            "waterlevel_75": "waterlevel_75",
+            "waterlevel_100": "waterlevel_100"
+        },
+		"min": {
```

Comment:
For min, max and palette: they are all the same, can you make a var from this and reuse? (if ever changes, makes life easier)

Reply:
True, they are all the same for now. it could be simplified, but then we have no control over defaults for any specific band.